### ISSUE TITLE
Can someone please tell me how to add a song?

### ISSUE THREAD

**Issue by KindaGayToMeTho** [ShadowMario/FNF-PsychEngine#216]
I've been trying to port Mid fight masses to the psych engine but I can't get the songs to work, can someone help me out here?

**Comment by oScoped** [comment]
Yeah. For the downloaded build. NOT THE SOURCE CODE. Go into mods/songs and make a new folder which has the same name as your song. Inside that folder put the Inst and Voices. INST HAS TO BE NAMED Inst and VOICES HAS TO BE NAMED Voices.

**Comment by aflacc** [comment]
check wiki
